crates/xilem_masonry/src/view/flex.rs

```diff
@@ -0,0 +1,120 @@
+use std::marker::PhantomData;
+
+use masonry::{
+    widget::{self, WidgetMut},
+    Widget, WidgetPod,
+};
+
+use crate::{ElementSplice, MasonryView, VecSplice, ViewSequence};
+
+// TODO: Allow configuring flex properties. I think this actually needs its own view trait?
```

Comment:
These kinds of specialised view traits would also be the idiomatic way to create and manage the properties of winit `Window`s, for example (in addition to menus?)